docs/Architecture.rst

```diff
@@ -3,18 +3,22 @@ Architecture
 
 .. image:: rocket2arch.png
 
-**What is the ``db`` module?** The database ``db`` module consists of
-the ``facade`` and the ``dynamodb`` database we are using.
+Our Flask server serves to handle all incoming Slack events, slash command, and
+Github webhooks. Slash commands are handled by :py:mod:`app.controller.command.parser`,
+and the remaining events and webhooks are handled by :py:mod:`app.controller.webhook.github.core`
+and :py:mod:`app.controller.webhook.slack.core`.
 
-**What is the ``command`` module?** The ``command`` module is where the
-slack commands get parsed and passed on to the backend so models can be
-created and the database can be populated.
+We store our data in an Amazon DynamoDB, which can be accessed directly by the
+database facade :py:class:`db.dynamodb.DynamoDB`.
 
-**What is the ``model`` module?** The ``model`` module is where models
-are constructed. Currently we have ``Team`` and ``User`` models.
+We treat Github itself as the sole source of truth. Any modifications done on the
```

Comment:
```suggestion
We treat GitHub itself as the sole source of truth. Any modifications done on the
```

etc - though we can also have a separate PR to make all these fixes in one go

Reference: official branding

![image](https://user-images.githubusercontent.com/23356519/95804617-cbe2d380-0d35-11eb-8938-4b836a2ce14d.png)